lib/babe/build.go

```diff
@@ -26,7 +26,7 @@ const (
 
 // construct a block for this slot with the given parent
 func (b *Service) buildBlock(parent *types.Header, slot Slot, rt runtime.Instance,
-	authorityIndex uint32, proof *VrfOutputAndProof) (*types.Block, error) {
+	authorityIndex uint32, proof *VrfOutputAndProof, ifPrimary bool) (*types.Block, error) {
```

Comment:
```suggestion
	authorityIndex uint32, proof *VrfOutputAndProof, secondarySlots byte) (*types.Block, error) {
```

since `types.ConfigData.SecondarySlots` is type `byte` allowing us to indicate primary, secondary plain, or secondary VRF, it would be good to pass that in here so that when we implement secondary VRF we can just check the byte